wasm/datatypes/valtype.py

```diff
@@ -101,3 +113,34 @@ def get_integer_type(cls, num_bits: BitSize) -> 'ValType':
             raise ValueError(
                 f"Invalid bit size.  Must be 32 or 64: Got {num_bits}"
             )
+
+    def validate_arg(self, arg):
+        if self is self.i32:
+            bounds = (0, constants.UINT32_MAX)
+            type_ = int
+        elif self is self.i64:
+            bounds = (0, constants.UINT64_MAX)
+            type_ = int
+        elif self in {self.f32, self.f64}:
+            # TODO: proper bounds
+            bounds = (sys.float_info.min, sys.float_info.max)
```

Comment:
If this isn't hard to track down in the spec, seems like worth adding now.

Reply:
I'm going to address it as my next task when I properly implement the floating point operations.  I tried to track down but it turns out that you have to pay money to get a copy of the IEEE-754 floating point standard: https://standards.ieee.org/standard/754-2008.html